packages/extension-api/src/extension-api.d.ts

```diff
@@ -1385,4 +1385,51 @@ declare module '@podman-desktop/api' {
     export function saveImage(engineId: string, id: string, filename: string): Promise<void>;
     export const onEvent: Event<ContainerJSONEvent>;
   }
+
+  export interface AccountInfo {
+    id?: string;
+    label: string;
+  }
+
+  export interface AuthenticationSession {
+    idToken: string | undefined;
+    readonly id: string;
+    readonly accessToken: string;
+    readonly scopes: ReadonlyArray<string>;
+    readonly account: AccountInfo;
```

Comment:
```suggestion
    readonly account: AuthenticationSessionAccountInformation;
```

Reply:
Should I just copy full authentication API from vscode?

Reply:
would be easier